training/1_populate_table/populate-table.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @example
+ * // returns a table with filled data from the passed array
+ * populateTable ([
+ * {
+ *   id: 1,
+ *   name: 'Andrii',
+ *   age: 22,
+ *   isWorkExperience: true
+ * },
+ * {
+ *   id: 2,
+ *   name: 'Anna',
+ *   age: 21,
+ *   isWorkExperience: false
+ * },
+ * {
+ *   id: 3,
+ *   name: 'Marta',
+ *   age: 23,
+ *   isWorkExperience: true
+ *  },
+ * ])
+ * function populateTable takes array of objects and builds the table
+ * @param {Array} arr, array of objects
+ */
+function populateTable(arr) {
+  let tableString = '<table>';
+  const body = document.getElementsByTagName('body')[0];
```

Comment:
It is better not to build DOM elements as a  concatenation of strings but using `createElement` and `appendChild`